databricks_sql_connector_core/src/databricks/sql/client.py

```diff
@@ -22,6 +27,7 @@
     ParamEscaper,
     inject_parameters,
     transform_paramstyle,
+    ArrowQueue,
```

Comment:
Is this import used anywhere in this file? If not - please remove it

Reply:
fixed